flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/nfa/ComputationState.java

```diff
@@ -74,11 +80,35 @@ public long getStartTimestamp() {
 		return state;
 	}
 
+	public State<T> getPreviousState() {
+		return previousState;
+	}
+
 	public T getEvent() {
 		return event;
 	}
 
 	public DeweyNumber getVersion() {
 		return version;
 	}
+
+	public static <T> ComputationState<T> createStartState(final State<T> state) {
+		Preconditions.checkArgument(state.isStart());
+		return new ComputationState<>(state, null, null, -1L, new DeweyNumber(1), -1L);
+	}
+
+	public static <T> ComputationState<T> createStartState(final State<T> state, final DeweyNumber version) {
+		Preconditions.checkArgument(state.isStart());
+		return new ComputationState<>(state, null, null, -1L, version, -1L);
+	}
+
```

Comment:
Indent the arguments to separate them from the body of the method. This will make the code easier to read.